include/inviwo/core/util/filesystem.h

```diff
@@ -154,6 +154,11 @@ IVW_CORE_API std::string getWorkingDirectory();
  */
 IVW_CORE_API std::string getExecutablePath();
 
+
+IVW_CORE_API std::string getInviwoBinDir();
```

Comment:
Is this not:
```suggestion
IVW_CORE_API std::string getExecutableDir();
```

Reply:
No. not when the inviwo is used as a library. As when used from python. Then the exe is python.exe... Most uses of getExecutableDir should probably be removed/changed. 

Im using the directory of the inviwo-core dll, which is a bit better at least. But that might also run into problems in some cases I think.